packages/iTwinUI-react/src/core/Table/TablePaginator.tsx

```diff
@@ -282,7 +283,15 @@ export const TablePaginator = (props: TablePaginatorProps) => {
       ref={paginatorResizeRef}
       {...rest}
     >
-      <div className='iui-left' />
+      <div className='iui-left'>
+        {totalSelectedRowsCount > 0 && (
+          <span className='iui-table-paginator-selection-count'>
+            {`${totalSelectedRowsCount} ${
+              totalSelectedRowsCount > 1 ? 'rows' : 'row'
```

Comment:
```suggestion
              totalSelectedRowsCount === 1 ? 'row' : 'rows'
```
nit: Maybe a more safer boolean could be the one above?

If in the future we want to show even this count when 0 rows are selected, we don't need to change this boolean.